### PR TITLE
release notes 1.5: copy over from https://hackmd.io/f3yFZBP3ScO_BsaPn…

### PR DESCRIPTION
**Pull request by maelvls**
Jake and I have wrote a draft in https://hackmd.io/f3yFZBP3ScO_BsaPnscLFA. This PR contains the result of our work.

Related (internal):

- https://github.com/jetstack/jetstack-website/pull/217
- https://github.com/jetstack/jetstack-website/pull/218

### CODE REVIEW

File: content/en/docs/release-notes/release-notes-1.5.md [hunk 1]
```diff
@@ -7,60 +7,291 @@ type: "docs"
...
+ready to be used:
 
-We introduce the concept of `SecretTemplate` for `Certificates`. When a certificate is issued, a new `Secret` is created to hold the certificate data. This secret is created by cert-manager. In order to use third-party solutions such as [kubed](https://github.com/kubeops/kubed) to copy this secret to multiple namespaces, this secret must be annotated.
+```shell
+kubectl cert-manager check api
+```
 
-`SecretTemplate` is optional. Labels and annotations from the template will be synced to the `Secret` at the time when the certificate is created or renewed.
+The plugin also learned to guess the version of cert-manager running on your
+cluster, similarly as to `kubectl version`:
```
Comment (wallrj) [inline content/en/docs/release-notes/release-notes-1.5.md:139]:
```suggestion
The plugin also learned to discover the version of cert-manager running on your
cluster, similarly as to `kubectl version`:
```

File: content/en/docs/release-notes/release-notes-1.5.md [hunk 2]
```diff
@@ -7,60 +7,291 @@ type: "docs"
...
 
-## Other Changes
+### Other (Cleanup or Flake)
 
-### TODO
+- A conformance end-to-end testing suite was added for the
+  CertificateSigningRequest resources
+  ([#4101](https://github.com/jetstack/cert-manager/pull/4101)).
+- Reduce binary sizes from 74MB down to 49MB by adding the Go ldflag `-w`
+  ([#4181](https://github.com/jetstack/cert-manager/pull/4181).
```
Comment (wallrj) [inline content/en/docs/release-notes/release-notes-1.5.md:297]:
```suggestion
  ([#4181](https://github.com/jetstack/cert-manager/pull/4181)).
```

**Review by wallrj — CHANGES_REQUESTED** [review]
Nice overview. I spotted couple of typos.

**Comment by SgtCoDFish** [comment]
There are a bunch of legitimate test failures in `pull-cert-manager-website-verify` - looks like `.spelling` needs updating, which I'll do now :+1:

File: content/en/docs/release-notes/release-notes-1.5.md (hunk 1 above)
Comment (SgtCoDFish) [reply content/en/docs/release-notes/release-notes-1.5.md]: I've just implemented this :+1:

File: content/en/docs/release-notes/release-notes-1.5.md (hunk 2 above)
Comment (SgtCoDFish) [reply content/en/docs/release-notes/release-notes-1.5.md]: I've just implemented this :+1:

**Review by wallrj — APPROVED** [review]
/lgtm

**Review by wallrj — COMMENTED** [review]
Consider moving the SecretTemplate feature to the headlines since it was an external contribution and much requested.

**Comment by maelvls** [comment]
> Consider moving the SecretTemplate feature to the headlines since it was an external contribution and much requested.

Done! I moved Irbe's annotations PR and `secretTemplate` to a new section. Does it make sense?

**Review by wallrj — APPROVED** [review]
Thanks @maelvls 

There was a test flake so I've added retest.

/lgtm

**Pull request merged**